app/views/allocations/_case_information.html.erb

```diff
@@ -22,9 +22,17 @@
     </td>
   </tr>
   <tr id="oasys-date" class="govuk-table__row">
-    <td class="govuk-table__cell">Last completed layer 3 OASys</td>
+    <td class="govuk-table__cell">Last completed OASys</td>
     <td class="govuk-table__cell table_cell__left_align">
-      <%= format_date(@oasys_assessment, replacement: 'This prisoner has not had a layer 3 OASys assessment.') %>
+      <% if @oasys_assessment.nil? %>
+        No OASys information for this prisoner
+        <%  %>
+      <% elsif @oasys_assessment.fetch(:assessment_type) == Faraday::ConflictError %>
+        This prisoner has more than one OASys record linked to their prison number. Merge these records or delete
+        one so we can display their OASys information.
+      <% else %>
+        <%= "Layer #{@oasys_assessment.fetch(:assessment_type)[-1]} – #{format_date(@oasys_assessment.fetch(:completed))}" %>
```

Comment:
```suggestion
        Layer <%= @oasys_assessment.fetch(:assessment_type)[-1] %> – <%= format_date(@oasys_assessment.fetch(:completed)) %>
```

This might be slightly nicer to read.